recipes/escape/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class EscapeConan(ConanFile):
+    name = "escape"
+    description = "Terminal Escape Sequence Library in C++17"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/a-n-t-h-o-n-y/Escape"
+    topics = ("escape sequence", "mouse", "tui")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "clang": "7",
+            "apple-clang": "12",
+            "Visual Studio": "16",
+            "msvc": "192",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
```

Comment:
```suggestion
            )
        if self.setting.build_type == "Debug":
            raise ConanInvalidConfiguration(f"{self.ref} does not support debug installation.")
```

The upstream only installs Release as build type: https://github.com/a-n-t-h-o-n-y/Escape/blob/main/CMakeLists.txt#L58

I don't think you need to patch it, instead, just raise an error. I'll open a PR to the upstream asking to change it.